app/src/main/java/de/danoeh/antennapod/ui/episodeslist/FeedItemMenuHandler.java

```diff
@@ -225,6 +229,8 @@ public static boolean onMenuItemClicked(@NonNull Fragment fragment, int menuItem
             DBWriter.markItemPlayed(selectedItem, FeedItem.UNPLAYED, true);
         } else if (menuItemId == R.id.visit_website_item) {
             IntentUtils.openInBrowser(context, selectedItem.getLinkWithFallback());
+        } else if (menuItemId == R.id.visit_social_interest_website) {
+            IntentUtils.openInBrowser(context, selectedItem.getPodcastIndexSocialUrl());
```

Comment:
Please add a warning message here ("you are leaving AntennaPod and visiting http://xy") to increase the chances that this won't get AntennaPod blocked on Google Play

Reply:
done with adding warning dialog